### DISCUSSION TITLE
How do I incorporate multiple bib files in quarto books

### DISCUSSION THREAD

**Discussion by abduazizR** [quarto-dev/quarto-cli#9686] [Q&A]
### Description

Hi 

I am currently working on a quarto book. The issue is that the only bibliography the project recognizes is the one generated in main directory of the book folder. So, if I have .qmd files inside a folder called "Chapters", rendering those files generate .bib files that are not recognized by the _quarto.yml file.

Is there a way to solve this?

Thanks

**Comment by mcanouil** [comment]
`bibliography` takes file paths. You should be able to give bib file from anywhere within your Quarto project.

Could you share a small concrete example of your setup?

**Reply by abduazizR** [reply]
Thanks for the quick response! I am running this repo https://github.com/kelly-sovacool/rackham/tree/main in particular

**Reply by abduazizR** [reply]
I tried to put chapters/references.bib 

but it does not work. Maybe I am doing the indentation incorrectly. Could you please show me an example of how to set this up correctly?

Thanks

**Reply by mcanouil** [reply]
Sorry, but I don't see the link with what you initially described.
You have only one bib file `references.bib` that you provided to `bibliography` at a project level.

**Reply by abduazizR** [reply]
Oh yes yes
The original repo doesn't have citations inside the chapters folder. I am asking if I were to put citations in .qmd files inside the chapters folder and a couple of `references.bib` are generated, how can I invoke them in `_quarto.yml`?

**Reply by mcanouil** [reply]
As I said and as documented `bibliography` takes paths.
So you need to provide the paths:

```yaml
bibliography:
  - whatever/path/to/file.bib
  - a/file/somewhere/else.bib
```
